src/lib/ui/Popup.js

```diff
@@ -126,64 +128,30 @@ Popup.prototype = {
     }
     this.app.broker.publish('popup::popupClose');
   },
-  _getImageURLForResolution: function(aWin, aURL, aWidth, aHeight) {
-    if (!aURL.endsWith('.ico') && !aURL.endsWith('.ICO')) {
-      return aURL;
-    }
-    const width = Math.round(aWidth * aWin.devicePixelRatio);
-    const height = Math.round(aHeight * aWin.devicePixelRatio);
-    return aURL + (aURL.contains('#') ? '&' : '#') +
-           '-moz-resolution=' + width + ',' + height;
-  },
-  // TODO: use UnifiedComplete instead of calling these services separately (#114)
   _appendCurrentResult: function() {
-    const autocompleteResults = this._getAutocompleteSearchResults();
-    if (this.inPrivateContext) {
-      this.app.broker.publish('popup::autocompleteSearchResults', autocompleteResults);
-      this.app.broker.publish('popup::suggestedSearchResults', []);
-    } else {
-      this._getSearchSuggestions().then((searchSuggestions) => {
-        this.app.broker.publish('popup::autocompleteSearchResults', autocompleteResults);
-
-        delete searchSuggestions.formHistoryResult;
-        this.app.broker.publish('popup::suggestedSearchResults',
-                                 searchSuggestions);
-      }, (err) => {
-        Cu.reportError(err);
-        this.app.broker.publish('popup::autocompleteSearchResults', autocompleteResults);
+    let placesResults;
```

Comment:
This part of the diff looks pretty nuts, but mostly because of indentation changes.